src/features/filter/Filter.tsx

```diff
@@ -37,8 +37,7 @@ export const Filter = () => {
     items: classFilterArr.map(([key, value]) => {
       return {
         key,
-        label: <Checkbox checked={value}>{key}</Checkbox>,
-        onClick: handleFilterChange(key, value),
+        label: <Checkbox checked={value} onChange={handleFilterChange(key, value)}>{key}</Checkbox>,
```

Comment:
This solution doesn't work for click in empty space in the row. 
I had tried different options and couldn't find anything. Might we should replace Dropdown with Popover and use Checkbox Group instead of Menu Items?
<img width="293" alt="Screenshot 2023-02-07 at 18 46 46" src="https://user-images.githubusercontent.com/59090607/217305234-71a613e2-ee59-4241-9df6-25273f3f5bc1.png">

Reply:
thank you!
but I just made the ckickable area row-width. please, check out `src\features\filter\styles\filter.less`